README.rst

```diff
@@ -123,43 +118,42 @@ server is started.
         pytest . --ansys-path "C:\Program Files\Ansys Inc\v232"
 
 
-#.  Run tests with a local gRPC server executable:
```

Comment:
Remove local gRPC server option because it does not work anymore.

Reply:
What's the recommended way of testing with a local dev build, then?

Reply:
Yes. I think the easiest would be to run a customer portal build with a custom tag. Another alternative would be to just copy the composite_operators.dll from the dpf_composites bin folder into the installer and then using the server from the installer. I have not yet tested these two workflows yet. For context: The workflow with the dpf server exe was broken very often due to internal changes in the dpf package which are often difficult to debug.